### ISSUE TITLE
[Bug] Ruby Parsing issue

### ISSUE THREAD

**Issue by ankit-privado** [joernio/joern#3040]
Describe the bug

Ruby parser fails for the below code
Please keep the conversation English! Make it easy for us and share your input code or a minified example.

To Reproduce
Load [this](https://github.com/odinsride/olubalance/blob/6fe35f3a84ab25dc0b6822f96a3a660174cace00/app/controllers/transactions_controller.rb#L8) code in the Ruby parser and see it failing.

Expected behavior
Parsing should pass

Screenshots
NA

Desktop (please complete the following information):
NA

Additional context
NA

**Comment by ankit-privado** [comment]
@xavierpinho you can also refer this sample code: https://github.com/zinagithub/blogger/blob/23287b51baa0ea3c1758334a6d5fb83611e057cf/config/spring.rb#L2

**Comment by ankit-privado** [comment]
@xavierpinho Would it be appropriate to close this issue and create separate tickets for all Ruby % shortcuts? This will allow anyone to work on each task individually.

**Comment by xavierpinho** [comment]
> @xavierpinho Would it be appropriate to close this issue and create separate tickets for all Ruby % shortcuts? This will allow anyone to work on each task individually.

I prefer to give this one preference and close the others by mentioning they are duplicates of this one. What matters are code samples, and they are not lost that way.

**Comment by ankit-privado** [comment]
yes sure, not yet separated. thanks

**Comment by pandurangpatil** [comment]
@xavierpinho what items are pending on this ticket?

**Comment by xavierpinho** [comment]
@pandurangpatil I believe all except `%s` are supported in the `ruby-dev` branch at the moment. `%s` is (mutatis mutandis) the same as `%w`, I just didn't notice it was missing until yesterday.
